src/github.com/getlantern/flashlight/proxiedsites/proxiedsites.go

```diff
@@ -25,10 +26,13 @@ var (
 	startMutex sync.Mutex
 )
 
-func Configure(cfg *proxiedsites.Config, proxyAddr string) {
-	delta := proxiedsites.Configure(cfg, proxyAddr)
+func Configure(cfg *proxiedsites.Config) {
+	delta := proxiedsites.Configure(cfg)
 	startMutex.Lock()
 
+	if delta != nil {
+		updateDetour(delta)
+	}
 	if service == nil {
```

Comment:
We may remove the chunks of code to maintain custom whitelist at all, but I choose not to touch them now.